keyboards/oktiboard/readme.md

```diff
@@ -0,0 +1,11 @@
+# oktiboard
+
+* Keyboard Maintainer: QMK Community
+* Hardware Supported: Pro Micro
+    * 16 NeoPixel Led
+
+Make example for this keyboard (after setting up your build environment):
+
+make oktiboard:default
```

Comment:
```suggestion
    make oktiboard:default
```

Reply:
Line 9 should lead with four spaces, as in fauxpark's comment.